modules/sovrnBidAdapter.js

```diff
@@ -63,7 +64,17 @@ export const spec = {
           },
           tagid: String(utils.getBidIdParameter('tagid', bid.params)),
           bidfloor: utils.getBidIdParameter('bidfloor', bid.params)
-        });
+        }
+
+        sovrnImps.push(imp);
```

Comment:
it does not actually matter but I would prefer to see the push after the variable is complete.

Reply:
Moved to the right place, thank you!